.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: python
+
+sudo: false
+
+matrix:
+  fast_finish: true
+  include:
+  - python: 3.6
+    env: TEST_TARGET=default
+  - python: 3.6
+    env: TEST_TARGET=coding_standards
+  allow_failures:
+  - python: 3.6
+    env: TEST_TARGET=coding_standards
+
+before_install:
+    - wget http://bit.ly/miniconda -O miniconda.sh
+    - bash miniconda.sh -b -p $HOME/miniconda
+    - export PATH="$HOME/miniconda/bin:$PATH"
+    - conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
+    - conda update conda
+    - conda create --yes -n TEST python=$TRAVIS_PYTHON_VERSION --file requirements.txt --file requirements-dev.txt
+    - source activate TEST
```

Comment:
I am using conda packages for the testing and the failures may be mismatches due to the underlying `blas`.
In this case I believe it is `mkl`. We can try `openblas` later.